src/main/java/me/jellysquid/mods/sodium/client/world/SodiumChunkManager.java

```diff
@@ -60,6 +61,9 @@ public void unload(int x, int z) {
         if (unloadedChunk != null) {
             this.world.unloadBlockEntities(unloadedChunk);
             this.onChunkUnloaded(x, z);
+
+            // Invoke Fabric API's ClientChunkEvents
+            ClientChunkEvents.CHUNK_UNLOAD.invoker().onChunkUnload(this.world, unloadedChunk);
```

Comment:
As said by @Leo40Git you should not directly depend on FAPI and have a similar system to #546.

Reply:
You probably mean #545 (that's what Leo said earlier), as this PR is #546

Reply:
Typo moment, oops